### DISCUSSION TITLE
The Motorola 68000 - Powering the Gaming Revolution - Retro Reversing (Reverse Engineering)

### DISCUSSION THREAD

**Discussion by giscus** [RetroReversing/retroReversing#144] [Post Comments]
# The Motorola 68000 - Powering the Gaming Revolution - Retro Reversing (Reverse Engineering)

RetroReversing, the home of retro game reverse engineering. Get resources, tutorials, and tools for analyzing video games and join the community today!

https://www.retroreversing.com/68k

**Comment by ghost** [comment]
Who told you the 68000 was a RISC CPU? It is not. The concept & term "RISC" didn't even exist in 1979




1

**Reply by RetroGameDeveloper** [reply]
Thank you, this is embarrassing I seem to have been reading about the 88k not the 68k 🤦 . This is exactly why I need to be adding references. Your correction is much appreciated, don’t want to be spreading false information.
